SharePointFramework/ProjectWebParts/src/data/SPDataAdapter/index.ts

```diff
@@ -175,8 +176,18 @@ class SPDataAdapter extends SPDataAdapterBase<ISPDataAdapterConfiguration> {
             break
           case 'TaxonomyFieldTypeMulti':
             {
-              if (syncToProject) {
-                // TODO: See SyncProjectModal TODO
+              if (syncToProject && fldValueTxt !== '') {
```

Comment:
We should perhaps fix this before merging @Remi749?

Reply:
I have tried, I can't figure it out. It's time sensitive that I get this out. So if we could look into that another time, that would be great.

Reply:
This is to be included in a pre-release for certain customer.